addons/website/static/src/scss/primary_variables.scss

```diff
@@ -1937,35 +2035,48 @@ $o-theme-color-palettes-compatibility-indexes: null; // Default to the color one
 // Create the list of palettes proposed by the UI
 $o-selected-color-palettes-names: (
     // Proposed by the configurator
-    'default-1',
-    'default-2',
-    'default-3',
-    'default-4',
-    'default-5',
+    'default-light-1',
+    'default-light-2',
+    'default-light-4',
+    'default-light-3',
+    'default-light-5',
+    'default-24',
+    'default-light-7',
+    'default-light-6',
+    'default-light-11',
+    'default-light-14',
+    'default-light-8',
     'default-6',
     'default-7',
     'default-8',
     'default-9',
+    'default-23',
+    'default-25',
+    'default-12',
```

Comment:
I don't think the order here has any importance. If you used the same as `PALETTE_NAMES`, it could make sense but you did not: I'd say just order them by name here.

Reply:
In fact, this would allow them to be organized by hue and saturation in the same way as in the Web configurator. This would ensure a certain consistency between them. :)
| Configurator | Editor |
|--------|--------|
| ![image](https://github.com/user-attachments/assets/b2c48a98-7ede-45aa-8693-74b48c02cdf9) | ![image](https://github.com/user-attachments/assets/e9a8c1a3-514d-4ce4-b506-8917492a764d) |

Reply:
Oh ok, my bad, indeed. But then why not using the exact same order as `PALETTE_NAMES` (used for the configurator)? (actually wondering why we have 2 different lists at all :shrug:)

Reply:
The Web configurator provides less color palettes than the Web editor.
We prefer to display a limited number of palettes when the user starts to create his website with the Configurator, to make it quick and simple and to prevent him from being overwhelmed by a large number of palettes. Once the website has been created, the user will have access to all the default palettes in the web editor 👍 .

Reply:
Ok, make sense. But from a technical point of view we'll have to improve (but it's not a problem created by this PR). We can definitely have only one entry for the palette list and mark each palette to be shown in the configurator and/or the builder, with an unique order. That will be for later. In the end, when the palettes will be in DB, that will be solved automatically.

Fine like this for now :+1: